test_Arkadi_Safaryan.py

```diff
@@ -0,0 +1,2 @@
+def test_print_name (name = 'Arkadi', surname = 'Safaryan'):
```

Comment:
Change your function and print also your favorite car model.

Reply:
I changed my func
Should i open new pull request?

Reply:
No need to open a new one.  Great, will merge.